package.json

```diff
@@ -34,9 +34,10 @@
         "@nomicfoundation/hardhat-chai-matchers": "^2.0.8",
         "@nomicfoundation/hardhat-network-helpers": "^1.0.12",
         "@nomiclabs/hardhat-solhint": "^4.0.1",
-        "@openzeppelin/contracts": "4.8.2",
-        "@openzeppelin/contracts-upgradeable": "4.8.2",
+        "@openzeppelin/contracts4": "npm:@openzeppelin/contracts@4.8.2",
+        "@openzeppelin/contracts-upgradeable4": "npm:@openzeppelin/contracts-upgradeable@4.8.2",
```

Comment:
double check that `upgrades` from `hardhat` uses the correct version as well.
This applies to `1_createGenesis.ts` and almost all the tooling

Reply:
Checked!